docs/how-to-guides/build-directory.rst

```diff
@@ -0,0 +1,19 @@
+.. SPDX-FileCopyrightText: 2023 The meson-python developers
+..
+.. SPDX-License-Identifier: MIT
+
+.. _how-to-guides-build-directory:
+
+*******************************
+Use a permanent build directory
+*******************************
+
+
+.. todo::
+
+   - Explain why you'd want to use the ``builddir`` option
+     - Mention it works properly when using ``--cross-file`` (eg. https://github.com/scipy/scipy/blob/1c836efe5ff37ffa4490756269b060a464690e62/.github/workflows/wheels.yml#L180-L188)
+   - Explain how ``builddir`` works
+   - Warn that user-configured build directories are not "use at your own risk"
```

Comment:
I think the intent here was "are" rather tha" are not" (at your own risk)? That said, I'd rephrase as "Explain the constraints that the user must adhere to when using a user-configured build directory". That also takes care of the second warning the line below.

Reply:
Yes. Thank you for catching that.